docs/docsource/operation/Zenko_CLI/index.rst

```diff
@@ -101,7 +100,70 @@ AWS gateway.
 
 Zenko can now respond to the set of S3 commands documented in the
 :version-ref:`Zenko Reference
-<https://documentation.scality.com/Zenko/{version}/reference/index.html>`.
+<https://documentation.scality.com/Zenko/{version}/reference/index.html>`.       
+
+Set Up Blobserver API Access
```

Comment:
Perhaps a note here that setting up HTTPS is required for using the SRP API with `az`